doc/ssl/SSL_CTX_set_ct_validation_callback.pod

```diff
@@ -2,39 +2,87 @@
 
 =head1 NAME
 
+SSL_ct_enable, SSL_CTX_ct_enable, SSL_ct_disable, SSL_CTX_ct_disable,
 SSL_set_ct_validation_callback, SSL_CTX_set_ct_validation_callback,
-SSL_get_ct_validation_callback, SSL_CTX_get_ct_validation_callback -
+SSL_ct_is_enabled, SSL_CTX_ct_is_enabled -
 control Certificate Transparency policy
 
 =head1 SYNOPSIS
 
  #include <openssl/ssl.h>
 
- int SSL_set_ct_validation_callback(SSL *s, ct_validation_cb callback, void *arg);
- int SSL_CTX_set_ct_validation_callback(SSL_CTX *ctx, ct_validation_cb callback, void *arg);
- ct_validation_cb SSL_get_ct_validation_callback(const SSL *s);
- ct_validation_cb SSL_CTX_get_ct_validation_callback(const SSL_CTX *ctx);
+ int SSL_ct_enable(SSL *s, int vtype);
+ int SSL_CTX_ct_enable(SSL_CTX *ctx, int vtype);
```

Comment:
Thanks, we agree on the interface.  While it makes sense for servers to provide two SCTs in case one of the logs is not supported by the client, I don't think it makes sense in general to require more than one valid SCT at the client end, indeed:
https://tools.ietf.org/html/draft-ietf-trans-rfc6962-bis-12#section-9.2
```
   To be considered compliant, a certificate MUST be accompanied by at
   least one valid SCT or at least one valid inclusion proof.  A
   certificate not accompanied by any valid SCTs or any valid inclusion
   proofs MUST NOT be considered compliant by TLS clients.  However,
   specifying the TLS clients' behavior once compliance or non-
   compliance has been determined (for example, whether a certificate
   should be rejected due to non-compliance) is outside the scope of
   this document.
```